mono/mini/interp/interp.c

```diff
@@ -1944,7 +1944,7 @@ interp_entry (InterpEntryData *data)
 	ThreadContext *context;
 	stackval result;
 	stackval *args;
-	stackval *retval;
+	stackval *retval = &result;
```

Comment:
This variable can just go away, and be replaced by `result` throughout the function.

Reply:
Will do in a separate PR and undo here.

Reply:
https://github.com/mono/mono/pull/18951